content/objects-and-math/studio/_index.md

```diff
@@ -0,0 +1,142 @@
+---
+title: "Studio: Objects and Math"
+date: 2023-08-31T13:16:22-05:00
+draft: false
+weight: 3
+originalAuthor: Sally Steuterman # to be set by page creator
+originalAuthorGitHub: gildedgardenia # to be set by page creator
+reviewer: John Woolbright # to be set by the page reviewer
+reviewerGitHub: jwoolbright23 # to be set by the page reviewer
+lastEditor: # update any time edits are made after review
+lastEditorGitHub: # update any time edits are made after review
+lastMod: # UPDATE ANY TIME CHANGES ARE MADE
+---
+
+In the exercises, you created objects to store data about the candidates for
+our animal astronaut corps. For this studio, we provide you with a ready-made
+set of candidates.
+
+You must create code to:
+
+1. Select the crew.
+1. Perform critical mission calculations.
+1. Determine the fuel required for launch.
+
+## Select the Crew
+
+To access the code for the first part, open `ObjectsStudio01.js` in `javascript-projects/objects-and-math/studio`.
```

Comment:
There's no reference to where a student can find the files mentioned in the Studio. Ideally, this should be added in a setup or getting started section in the book for all assignments, studios and exercises

Reply:
This is covered at the beginning of the book and why we had the students set up `javascript-projects` as part of prework. By this point in the class, it is appropriate to point students to where the files are using the path starting at `javascript-projects`. If you still feel this is too confusing, I would love to see a suggestion on what wording you think would make it better.

Reply:
Perhaps a reference link back is enough on this update? 12 chapters ago is a long time and we can't make any assumption since Operating System fail, new laptops may need to be used or Replit could be deployed

Reply:
Issue created to be picked up once the team has more capacity to implement link back references